app/code/Magento/Catalog/Test/Mftf/ActionGroup/AdminSelectAttributeSetOnEditProductPageActionGroup.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+ /**
+  * Copyright © Magento, Inc. All rights reserved.
+  * See COPYING.txt for license details.
+  */
+-->
+
+<actionGroups xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+              xsi:noNamespaceSchemaLocation="urn:magento:mftf:Test/etc/actionGroupSchema.xsd">
+    <actionGroup name="AdminSelectAttributeSetOnEditProductPageActionGroup">
+        <annotations>
+            <description>Selects the specified value from the Attribute Set dropdown.
+                         The Edit Product Page should be opened prior to Action Group execution</description>
```

Comment:
I would refactor this part. As for `The Edit Product Page should be opened prior to Action Group execution` sentence, I think that is extra. Usually, you are checking the ActionGroup implementation before using it.

```suggestion
            <description>Selects the specified value from the Attribute Set dropdown on the opened product edit page.</description>
```